src/app/devices/sol/sol.component.ts

```diff
@@ -0,0 +1,121 @@
+import { Component, OnInit, ViewEncapsulation, Input, Output, EventEmitter } from '@angular/core'
+import { Terminal } from 'xterm'
+import { AmtTerminal, AMTRedirector, TerminalDataProcessor, ConsoleLogger, Protocol, LogLevel } from 'ui-toolkit'
+import { ActivatedRoute } from '@angular/router'
+import { environment } from 'src/environments/environment'
+
+@Component({
+  selector: 'app-sol',
+  templateUrl: './sol.component.html',
+  styleUrls: ['./sol.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SolComponent implements OnInit {
+  @Input() solStatus: any = false
+  @Output() deviceStatus: EventEmitter<number> = new EventEmitter<number>()
+  public term: Terminal | any
+  public container!: HTMLElement | any
+  public terminal: AmtTerminal | any
+  public logger: ConsoleLogger = new ConsoleLogger(LogLevel.ERROR)
+  public redirector: AMTRedirector | any
+  public server: string = environment.mpsServer.replace('https://', '')
+  public uuid: string | any = this.route.snapshot.paramMap.get('id')
```

Comment:
this should be retrieved from activatedRoute in ngOnInit()

Reply:
done